docs/glossary.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Glossary
+sidebar_position: 90
+title: Glossary
+description: Kubewarden glossary
+keywords: [kubewarden, glossary]
+doc-persona: [kubewarden-all]
+doc-type: [explanation]
+doc-topic: [glossary]
+---
+
+<head>
+  <link rel="canonical" href="https://docs.kubewarden.io/glossary"/>
+</head>
+
+## A
+
+### AdmissionPolicy
+
+A namespace-wide resource. The policy processes only those requests targeting
+the namespace in which the AdmissionPolicy is defined.
+
+## C
+
+### ClusterAdmissionPolicy
+
+A ClusterAdmissionPolicy defines how policies evaluate requests.
```

Comment:
This is the same of the `AdmissionPolicy`. But this targets cluster-wide resources.